src/collection.js

```diff
@@ -3,25 +3,23 @@
 import BaseAdapter from "./adapters/base";
 import { waterfall, deepEqual } from "./utils";
 import { v4 as uuid4 } from "uuid";
-import { isUUID, pFinally } from "./utils";
+import { isUUID, pFinally, omitKeys } from "./utils";
 
-const RECORD_FIELDS_TO_CLEAN = ["_status", "last_modified"];
+const RECORD_FIELDS_TO_CLEAN = ["_status"];
 const AVAILABLE_HOOKS = ["incoming-changes"];
 
+
 /**
- * Cleans a record object, excluding passed keys.
- *
- * @param  {Object} record        The record object.
- * @param  {Array}  excludeFields The list of keys to exclude.
- * @return {Object}               A clean copy of source record object.
+ * Compare two records omitting local fields and synchronization
+ * attributes (like _status and last_modified)
+ * @param {Object} a    A record to compare.
+ * @param {Object} b    A record to compare.
+ * @return {boolean}
  */
-export function cleanRecord(record, excludeFields=RECORD_FIELDS_TO_CLEAN) {
```

Comment:
`cleanRecord()` was not documented, thus not officially exposed :]

I added some documentation about this change as part of the breaking changes for 3.X.

For [what I could see in the code](https://github.com/Kinto/kinto-admin/blob/42469960835bacabd7d904ee52015fd16c4a8109/scripts/reducers/form.js#L9) of kinto-admin, the function `omitKeys()` could be used instead (I would even suggest to copy its code instead of importing it from kinto.js)